projects/packages/jetpack-mu-wpcom/src/features/coming-soon/coming-soon.php

```diff
@@ -209,6 +209,7 @@ function disable_coming_soon_on_privacy_change( $old_value, $value ) {
  */
 function add_option_to_new_site( $blog_id, $user_id, $domain, $path, $network_id, $meta ) {
 	if ( 0 === $meta['public']
+		&& array_key_exists( 'options', $meta )
 		&& array_key_exists( 'wpcom_public_coming_soon', $meta['options'] )
```

Comment:
Pretty much the only reason to use `array_key_exists()` over `isset()` is when you want to tell the difference between null and unset. On the other hand, `isset()` lets you do like this
```suggestion
		&& isset( $meta['options']['wpcom_public_coming_soon'] )
```

Reply:
That is cleaner, thanks!